demo/index.html

```diff
@@ -1,5 +1,47 @@
 <html>
 <head>
+    <style>
```

Comment:
why are we including electron-44 (download changes) as part of crash reporter pr?